astlib/keyword.mli

```diff
@@ -0,0 +1,2 @@
+(** Check if a string is an OCaml keyword. *)
+    val is_keyword : string -> bool
```

Comment:
This probably shouldn't be indented!

Reply:
Good catch!

I'm glad that soon we'll be using ocamlformat!